velocity_manager:
  ros__parameters:
    max_velocity: 0.8  # 初始最大速度 0.8 m/s